### ISSUE TITLE
Donut does not display when certification hours exceeds required 40 hours.

### ISSUE THREAD

**Issue by paulagaube** [BCcampus/eypd#501]
### Description 

Donut does not display for a Learner when the total certification hours exceeds required 40 hours.

![learner-exceeds-certificate-no-donut](https://user-images.githubusercontent.com/26391359/42012341-86fb142c-7a4c-11e8-8178-2158f861917e.png)

**Expected behaviour** 

Full donut will show 40 hours completed. 

**Actual behaviour** 

No donut shows.

### Steps to reproduce the problem

As a logged-in Learner, on myEYPD page, I opened my Past Events and changed an event I had planned to attend from "Did Not Attend" to "Attended" using the radio button. Then clicked "Calculate My Hours". 

By confirming I attended this event in April, "Hopping Around",  I as given 123,456 certification hours (go big or go home).

No donut shows under the Event Summary (see image).

### System Information

Using Windows 7, Chrome 
This is the URL: https://eypd.bccampus.ca/members/paulatest/events/

- EYPD: Accessed June 27, 2018 at ~9 PM

**Comment by kathreenriel** [comment]
This is an important observation. What to do when the donut is full?

**Comment by mandily-p** [comment]
If user_hours >40, each hour represents 1/40th of the donut
Else each hour represents 360/total_hours.

Does that make sense?

**Comment by kathreenriel** [comment]
We need to consider the illustration or results that will be shown after a user exceeds 40 hours. @mandily-p the calculation you suggested may not take into account that several categories are also taken into account of the calculation currently presented in the legend.

**Comment by brettjforsyth** [comment]
This might have been fixed somewhere else as I can not reproduce it. I have included a screen shot of it working for 500+ hours but also tried it with +123,456 hours and it works. Checked on FF and chrome on a Mac.
![screen shot 2018-11-12 at 1 40 27 pm](https://user-images.githubusercontent.com/439832/48368062-a8faa100-e680-11e8-9edf-b2ba4bd5e011.png)

**Comment by mandily-p** [comment]
The container was previously on the page for the chart, it just didn't render properly. It may have just been a math error. In any case, it looks like it's working now! 👍

**Comment by paulagaube** [comment]
@brettjforsyth The colour wheel does display when I have completed 50/40 certification hours, which is great, but I noticed two things, which may be addressed in other issues: 
1. The colours are not consistent across all browsers (Windows 10 Firefox and Chrome are different than Edge, and also different than iPhone browsers Safari, Chrome and Firefox.) See image below.
2. Chrome and Firefox on Windows 10 display the hours to 8 decimal points (12.26666667). All other browsers display to one decimal point (12.3).  

<img width="800" alt="501-colourwheel-excess-hours" src="https://user-images.githubusercontent.com/26391359/49104727-885f4900-f233-11e8-8387-98fad48f33bd.png">

**Comment by paulagaube** [comment]
I am also going to add an event with an odd number of hours to see how the numbers render.

**Comment by brettjforsyth** [comment]
@paulagaube hmm.. it looks like the chrome and firefox screen shots are running the old code from your cache. Can you try clearing your cache and testing again. The colors still aren't set in the admin so it will look a bit weird for the time being.

**Comment by paulagaube** [comment]
@brettjforsyth Right! Thanks. The multiple decimal points in Chrome and Firefox disappeared after I cleared my cache. The donut is consistently displaying even if certification hours exceeds 40. 
@kathreenriel I believe this issue is DONE.

**Comment by kathreenriel** [comment]
Yes, done! Thanks Paula.

**Comment by kathreenriel** [comment]
checked this under Olivia's username this is works as expected.
